ui/pages/send/send-content/send-amount-row/send-amount-row.component.js

```diff
@@ -31,12 +31,14 @@ export default class SendAmountRow extends Component {
         onChange={this.handleChange}
         token={asset.details}
         value={amount}
+        primaryNumberOfDecimals={asset.decimals < 15 ? asset.decimals : 15}
```

Comment:
Should this be `asset.decimals <= 16 ? asset.decimals : 16`? I'm now second guessing my comments earlier — I'm not sure if the cutoff is 14, 15, or 16.